tiller_python_service/git_actions.py

```diff
@@ -10,6 +10,31 @@
 import re
 import openai
 
+
+def my_new_sorting_function(arr):
+    n = len(arr)
+    # optimize code, so if the array is already sorted, it doesn't need
```

Comment:
This function is implemented very inefficiently, can we replace it with a merge sort? help tiller